src/migrations/20201109053127-create-amenity.js

```diff
@@ -7,7 +7,7 @@ module.exports = {
         primaryKey: true,
         type: Sequelize.UUID
       },
-      AccommodationId: {
+      accommodationId: {
```

Comment:
@CYIMANA-Faisal create a new migration file that changes the name of this column